src/makers/midas/antennas.ts

```diff
@@ -0,0 +1,18 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const MIDAS_GATEWAY_US = {
+  name: 'MIDAS_GATEWAY(US 915)',
```

Comment:
@MidasWireless do you want your Hotspot to appear in a list with an underscore in the name?

Reply:
already modified hotspot names in antennas.ts and hotspots.ts